x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -91,6 +91,13 @@ export const EA_DOCS_RISK_USERS = i18n.translate(
   }
 );
 
+export const EA_DOCS_ENTITY_RISK_SCORE = i18n.translate(
+  'xpack.securitySolution.riskScore.riskScorePreview.eaDocsEntities',
+  {
+    defaultMessage: 'Entity risk score',
```

Comment:
According to the [issue](https://github.com/elastic/security-team/issues/7920) description, shouldn't this text be `How is the risk score calculated?`?

Reply:
Yeah my bad, nice catch! Done.